index.html

```diff
@@ -32,10 +32,15 @@ <h3> Next meetup </h3>
         <table><tbody>
         <tr><th><b>03.12.2019</br></th>
             <td>
-              <b>Location:</b> TBD</br>
+              <b>Location:</b> <a href=https://quantstack.net/index.html>Artelys</a> (81 rue Saint Lazare, 75009, Paris)</br>
+              <i>
+              Please fill-in <a href="https://docs.google.com/forms/d/e/1FAIpQLSeHt3aB81E8qAFVuR6wo8YejLSPjGjW_q_qWveQ9-d5xom0SQ/viewform?vc=0&c=0&w=1">this form</a> to register to the event.
+              </i></br>
               <b>Schedule:</b> 19h </br>
               <b>Topics:</b>
-              TBD
+              <p>
+                <a href="https://www.mpq.univ-paris-diderot.fr/?Nouvelle-traduction-1510-Filippo-Vicentini">Filippo</a> will give a talk.
```

Comment:
The amount of translation there is pretty small ... why not use `https://www.mpq.univ-paris-diderot.fr/?Filippo-Vicentini` directly?